core/src/test/java/fr/sncf/osrd/api/PathfindingTest.java

```diff
@@ -512,7 +512,7 @@ public void testMachingRouteOffsets(String path, boolean inverted) throws Except
     }
 
     /** Returns true if the response is a 400 */
-    private static boolean contains400(List<String> res) {
+    public static boolean contains400(List<String> res) {
```

Comment:
This will probably disappear with your next changes, but this is basically a method that does:
`assertListContainsString(List<String>, String)`, and should be written likeso in a Utils class for tests ;)